docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
 
 Now that Grafana knows how to notify us, it's time to set up an alert rule:
 
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Alert rules**.
-1. Click **+ New alert rule**.
-1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**. For **Folder** type `fundamentals` and in the box that appears, press **Create: fundamentals**.
+1. In Grafana's sidebar, expand the **Alerting** list (bell icon) and then click **Alert rules**.
+1. Click the **Create alert rule** button.
+1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**.
 1. For **Section 2**, find the **query A** box. Choose your Prometheus datasource and enter the same query that we used in our earlier panel: `sum(rate(tns_request_duration_seconds_count[5m])) by(route)`. Press **Run queries**. You should see some data in the graph.
-1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`
-1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
+1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`. **query C** should be deleted to make this work.
+1. In **Section 3**, Select **+ Add New** under **Folder** dropdown menu. Name the new folder `fundamentals` and press enter. This will create the folder needed. In **Evaluation Group** name it also `fundamentals` for now. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
 1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
-1. Click **Save and exit** at the top of the page.
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Notification policies**.
+1. Click **Save rule and exit** at the top of the page.
+1. Expand Grafana's sidebar, expand the **Alerting** (bell icon) node and then click **Notification policies**.
```

Comment:
```suggestion
1. In the sidebar, click **Notification policies**.
```

Reply:
> Shouldn't we make it more clear where the 'Notification Policies' is located? Assuming the **Alerting** node is expanded isn't quite so clear. Just in case someone opens the sidebar and doesn't have it expanded for some reason.

I was walking my way through the tutorial as I made edits, so at this particular point in the steps, the sidebar should still be open (not the case all the time during the tutorial). Also, at this point, we'll have users 3 or 4 times to click the menu icon to open the sidebar, so if they have managed to close it, they've learned that behaviour by now.